experimenter/tests/integration/nimbus/pages/browser/__init__.py

```diff
@@ -8,12 +8,12 @@
 
 
 class Browser:
-    def execute_script(self, *args, script=None, context=None):
+    def execute_async_script(self, *args, script=None, context=None):
         if "chrome" in context:
             with self.context(self.CONTEXT_CHROME):
-                return self.execute_script(script, *args)
+                return self.execute_async_script(script, *args)
         else:
-            return self.execute_script
+            return self.execute_async_script
```

Comment:
I do not understand how this is supposed to work

Reply:
In the example you gave, context="chrome" so we're in the other branch. From inspection, you would have to use this like:

```
result = Browser.execute_async_script()(script)
```